### PR TITLE
Refactored Previous code for manager for analyze, export and  Ingestion

### PR DESCRIPTION
**Pull request by SanjeevLakhwani**
- Created a new component to display steps
- Combined analyze and export into one file
- Created a new component for reviewing and submitting workflows
- Converted on class based components in the way to function based components

### CODE REVIEW

File: src/components/manager/ManagerWorkflowInterfaceContent.js [hunk 1]
```diff
@@ -0,0 +1,195 @@
...
+                Object.entries(s.workflows).flatMap(([action, workflowsByAction]) =>
+                    Object.entries(workflowsByAction).map(([id, v]) => ({
+                        ...v,
+                        id, // e.g. phenopacket_json, vcf_gz
+                        serviceID,
+                        action,
+                    }))
+                )
+            )
+    );
```
Comment (gsfk) [inline src/components/manager/ManagerWorkflowInterfaceContent.js:48]: the same ornate code now appears three times in the code base (it appears to begin at workflowsStateToPropsMixin() in proptTypes.js). Do we need it three times?

**Review by gsfk — COMMENTED** [review]
I only have a few line-by-line comments, see below. 

Some of the code throws React warnings about missing ids in `map()` calls, although from what I can tell this an issue with the original code, not the changes.

File: src/components/DataManagerContent.js [hunk 2]
```diff
@@ -40,30 +52,69 @@ class DataManagerContent extends Component {
...
+                        <div
+                            style={{
+                                padding: "24px",
+                                backgroundColor: "white",
+                            }}
+                        >
+                            <Skeleton active />
+                        </div>
+                    }
+                >
```
[the remark this answers is not in the document]
Comment (SanjeevLakhwani) [reply src/components/DataManagerContent.js]: changed it

File: src/components/manager/ManagerWorkflowInterfaceContent.js [hunk 3]
```diff
@@ -0,0 +1,195 @@
...
+import StepsTemplate from "./StepsTemplate";
+import WorkflowConfirmationForm, { FIELD_OPTIONS } from "./WorkflowConfirmationForm";
+
+// TODO: when redirected to this page from a Project/Dataset page,  doesn't defaults to selected Project/Dataset,
+// fix: Have the select box input the value and pass that along
+const ManagerWorkflowInterfaceContent = ({ managerType }) => {
+    const dispatch = useDispatch();
+    const history = useHistory();
+    const location = useLocation();
+    console.log("location", location);
```
[the remark this answers is not in the document]
Comment (SanjeevLakhwani) [reply src/components/manager/ManagerWorkflowInterfaceContent.js]: removed it

File: src/components/manager/ManagerWorkflowsContent.js [hunk 4]
```diff
@@ -1,31 +1,48 @@
...
+        );
     }
 }
 
 ManagerWorkflowsContent.propTypes = workflowsStateToPropsMixinPropTypes;
 
-const mapStateToProps = state => ({...workflowsStateToPropsMixin(state)});
+const mapStateToProps = (state) => ({ ...workflowsStateToPropsMixin(state) });
 
 export default connect(mapStateToProps)(ManagerWorkflowsContent);
```
[the remark this answers is not in the document]
Comment (SanjeevLakhwani) [reply src/components/manager/ManagerWorkflowsContent.js]: There are and in the new version it is now func based

File: src/components/manager/ManagerWorkflowInterfaceContent.js (hunk 1 above)
Comment (SanjeevLakhwani) [reply src/components/manager/ManagerWorkflowInterfaceContent.js]: changed it to a utility function

**Comment by ppillot** [comment]
During tests, the following action leads to a blank page:
- go to Admin/Data Manager
- click on the `Workflows` tab
Screen Shot 2022-09-30 at 10 23 31 AM
Screen Shot 2022-09-30 at 10 33 43 AM

**Comment by ppillot** [comment]
The descriptions for the table, data-type and project are wrong: on the internal index is printed on the page instead of the corresponding labels
Screen Shot 2022-10-07 at 11 20 41 AM

**Review by ppillot — CHANGES_REQUESTED** [review]
See the attached screenshots for the bugs uncovered during testing (workflows list and details for each workflow)

**Pull request closed without merging**